Eb genotype concordance should read gzipped files

eitanbanks opened the pull request:
(I may have messed up the git merging on this one...)

nh13 commented on src/java/picard/vcf/GenotypeConcordance.java:
```diff
@@ -140,8 +140,11 @@ public static void main(final String[] args) {
-            if ((!Tribble.indexFile(TRUTH_VCF).exists() || (!Tribble.indexFile(CALL_VCF).exists()))) {
-                errors.add("Index file(s) not found for one or both of the VCFs.  Note - if intervals are specified, the VCF files must be indexed.");
+            if (!indexExists(TRUTH_VCF)) {
+                errors.add("The index file was not found for the TRUTH VCF.  Note that if intervals are specified, the VCF files must be indexed.");
```
Why not check if intervals are specified here too?

eitanbanks replied:
Because Kylee is taking care of that in her commit.

eitanbanks commented:
@nh13 checks are still failing.  Is this leftover from my git idiocy?

nh13 commented on src/java/picard/vcf/GenotypeConcordance.java:
```diff
@@ -152,6 +155,15 @@ public static void main(final String[] args) {
+     * @return true if an index file exists, false otherwise
+     */
+    private boolean indexExists(final File vcf) {
+        return !Tribble.indexFile(vcf).exists() && !Tribble.tabixIndexFile(vcf).exists();
```
Isn't this logic backwards.  You want to return ```true``` if it exists so it should be ```Tribble.indexFile(vcf).exists() || !Tribble.tabixIndexFile(vcf).exists()```?

eitanbanks commented:
Okay @nh13 I tried to fix up my branch based on the git magic you taught me today, added back the fix for the logic, rebased, and re-pushed.
Did I do it all correctly?

nh13 commented:
:+1:
